.github/workflows/release.yml

```
@@ -14,16 +14,21 @@ jobs:
     name: release
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/create-github-app-token@v1
+      # build local version to create token
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: "npm"
+      - run: npm ci
+      - run: npm run build
+      - uses: .
```

```suggestion
      - uses: ./
```

This is how we do it in [test.yml](https://github.com/actions/create-github-app-token/blob/b94a2002247ecb9ffe877637ff04bf7bf536bac9/.github/workflows/test.yml#L38). I remember having trouble with the runner being very particular about relative paths in the past, but I'm not sure if this will be a problem or not in its current form.